.github/workflows/nightly-builds.yml

```diff
@@ -0,0 +1,119 @@
+name: Nightly Builds
+
+on:
+  pull_request:
+#  FIXME do not merge without removing this
+#  schedule:
+#    - cron: "0 0 * * *"
+
+jobs:
+
+  nightly-cluster-metrics-sigar:
+    name: Nightly Akka Cluster Metrics Test with Sigar
+    runs-on: ubuntu-20.04
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
+          fetch-depth: 0
+
+      - name: Set up JDK 8
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.8.0
+
+      - name: Cache Coursier cache
+        uses: coursier/cache-action@v6.2
+
+      - name: sbt akka-cluster-metrics/test
+        run: |-
+          sbt -jvm-opts .jvmopts-ci \
+            -Djava.security.egd=file:/dev/./urandom \
+            -Dakka.test.sigar=true \
+            -Dakka.cluster.assert=on \
+            clean akka-cluster-metrics/test
+
+      - name: Email on failure
+        if: ${{ failure() }}
+        uses: dawidd6/action-send-mail@v3
+        with:
+          server_address: smtp.gmail.com
+          server_port: 465
+          username: ${{secrets.MAIL_USERNAME}}
+          password: ${{secrets.MAIL_PASSWORD}}
+          subject: "Failed: ${{ github.workflow }} / ${{ github.job }}"
+          to: akka.official@gmail.com
+          from: Akka CI (GHActions)
+          body: |
+            Job ${{ github.job }} in workflow ${{ github.workflow }} of ${{github.repository}} failed!
+            https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}
+
+  jdk-nightly-tests:
+    name: JDK ${{ matrix.jdkVersion }} / Scala ${{ matrix.scalaVersion }}
```

Comment:
I guess this is ok for 2x2, but not sure we have to run all combinations when we expand this to let's say 3x3

Reply:
Yes, I agree. The description is up to date to compare the existing jobs vs. what is proposed by this PR.